src/components/Books.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { removeBook } from '../redux/books/books';
+import Form from './Form';
+
+const Books = () => {
+  const books = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+
+  return (
+    <div>
+      <ul>
+        {
+          books.map((book) => (
+            <li
+              key={book.id}
+            >
+              Book title:
+              {' '}
+              {book.title}
+              {' '}
+              -
+              Category:
+              {' '}
+              {book.category}
+              {' '}
+              -
+              {' '}
+              <button
+                type="button"
+                onClick={() => dispatch(removeBook(book))}
+              >
+                Remove book
+              </button>
+            </li>
+          ))
+        }
+      </ul>
+      <Form />
```

Comment:
- [ ] According to the project requirements, the `Books` component should render the `Book` component, but it seems that you do not have the `Book` component for now. Kindly let's create this component and render it inside the `Books` component 😉 
![image](https://user-images.githubusercontent.com/55465133/136262588-61b29959-9dce-4d2f-bea2-ac7f64a5012b.png)